### PR TITLE
[Typography] Fix SerializerAwareInterface description

### PR DESCRIPTION
**Pull request by idetox**
| Q             | A
| ------------- | ---
| Branch?       | 5.x
| Bug fix?      | no
| New feature?  | no 
| Deprecations? |no
| Tickets       |  -
| License       | MIT

Removed some unneeded comments in Serializer / Normalizer / Encoder directory

### CODE REVIEW

**Review by Nyholm — APPROVED** [review]
Im happy with this.. or remove the comment all together.

**Review by derrabus — APPROVED** [review]
This patch should be applied to the 3.4 branch, though.

File: src/Symfony/Component/Serializer/SerializerAwareInterface.php [hunk 1]
```diff
@@ -12,7 +12,7 @@
 namespace Symfony\Component\Serializer;
 
 /**
- * Defines the interface of encoders.
+ * Class accepting a serializer.
```
Comment (fabpot) [inline src/Symfony/Component/Serializer/SerializerAwareInterface.php]: I would remove the comment instead.

Comment (idetox) [reply src/Symfony/Component/Serializer/SerializerAwareInterface.php]: @fabpot would you remove the comment in NormalizerAwareInterface and DenormalizerAwareInterface aswell then ?

Comment (nicolas-grekas) [reply src/Symfony/Component/Serializer/SerializerAwareInterface.php]: If they don't provide significant value, yes.

Comment (idetox) [reply src/Symfony/Component/Serializer/SerializerAwareInterface.php]: Alright, expanded the removal to few other files

**Review by Nyholm — APPROVED** [review]
Thank you

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @idetox.

**Pull request merged**